packages/fauna/package.json

```diff
@@ -21,10 +21,19 @@
     "access": "public"
   },
   "scripts": {
-    "test": "exit 0"
+    "reset": "export FAUNA_ADMIN_KEY=secret;export FAUNADB_DOMAIN=localhost; export FAUNADB_SCHEME=http;export FAUNADB_PORT=8443;fauna-schema-migrate apply all",
+    "test:db:start": "docker run -d --rm --name faunadb -p 8443:8443 fauna/faunadb",
```

Comment:
Don't know what will happen if I already have a docker with same name running. Should we change the name to something else to avoid conflicts?

Reply:
Changed the container name to `next-auth-fauna-test`, thanks :wink: